engine/engine_test.go

```diff
@@ -0,0 +1,76 @@
+package engine
+
+import (
+	"testing"
+
+	"github.com/thrasher-corp/gocryptotrader/config"
+)
+
+func Test_loadConfigWithSettings(t *testing.T) {
+	empty := ""
+	somePath := "somePath"
+	tests := []struct {
+		name     string
+		flags    []string
+		settings *Settings
+		want     *string
+		wantErr  bool
+	}{
+		{
+			name:     "empty",
```

Comment:
This test relies on a user's real config. As that is what gets loaded. If said user has an encrypted config or a custom datadir set, this test will fail. I don't think this scenario is needed as a result
```
2020/09/11 10:15:14 Loading config file C:\Users\gloriousCode\AppData\Roaming\GoCryptoTrader\config.json..
    engine_test.go:64: loadConfigWithSettings() error = <nil>, wantErr true
```

Reply:
Ahh... true. I run my stuff in a clean environment so the user directory doesn't exist, the same is for for travis, but I can see that if the environment is not controlled, an existing config may exist in the default location.
Removed the case.